assets/app/scripts/directives/nav.js

```diff
@@ -28,11 +32,21 @@ angular.module('openshiftConsole')
         var select = $('.selectpicker', element);
 
         var updateOptions = function(projects) {
+          // Sort the projects by display name.
+          var projectArray = [];
           angular.forEach(projects, function(project) {
+            projectArray.push(project);
+          });
+          projectArray.sort(function(left, right) {
+            return displayName(left).localeCompare(displayName(right));
```

Comment:
Self answering yes: "Supported in the following document modes: Quirks, Internet Explorer 6 standards, Internet Explorer 7 standards, Internet Explorer 8 standards, Internet Explorer 9 standards, Internet Explorer 10 standards, Internet Explorer 11 standards. Also supported Store apps (Windows 8 and Windows Phone 8.1)."